app/bundles/LeadBundle/Tests/Form/Type/CampaignConditionLeadPageHitTypeTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\LeadBundle\Tests\Form\Type;
+
+use Mautic\CoreBundle\Test\AbstractMauticTestCase;
+use Mautic\LeadBundle\Form\Type\CampaignConditionLeadPageHitType;
+use PHPUnit\Framework\MockObject\MockObject;
+use Symfony\Component\Form\FormBuilderInterface;
+
+final class CampaignConditionLeadPageHitTypeTest extends AbstractMauticTestCase
+{
+    private CampaignConditionLeadPageHitType $campaignConditionPageHitType;
+    /** @var FormBuilderInterface&MockObject */
+    private FormBuilderInterface $formBuilderInterface;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->campaignConditionPageHitType = new CampaignConditionLeadPageHitType();
+        $this->formBuilderInterface         = $this->createMock(FormBuilderInterface::class);
+    }
+
+    public function testCampaignConditionPageHitTypeFormIsProperlyBuilt(): void
+    {
+        $this->formBuilderInterface->expects($this->exactly(4))
```

Comment:
Apologies I wasn't clearer in the request to test the form. I meant a test similar to this one:

https://github.com/mautic/mautic/blob/5.x/app/bundles/CampaignBundle/Tests/Controller/EventControllerFunctionalTest.php

Testing that the form has 4 fields has no value and it would have to be updated every time the form is changed. So it does not help with refactoring with confirming that everything still works but it adds more maintenance to it.